llvm/include/llvm/MC/MCPseudoProbe.h

```diff
@@ -240,12 +235,11 @@ class MCPseudoProbeInlineTreeBase {
   bool isRoot() const { return Guid == 0; }
   InlinedProbeTreeMap &getChildren() { return Children; }
   const InlinedProbeTreeMap &getChildren() const { return Children; }
-  std::vector<ProbeType> &getProbes() { return Probes; }
-  const std::vector<ProbeType> &getProbes() const { return Probes; }
-  void addProbes(ProbeType Probe) { Probes.push_back(Probe); }
+  ProbesType &getProbes() { return Probes; }
```

Comment:
Do you know where we require this function(anywhere can't replace with `const`), I thought we won't change the probe after it's decoded. is it possible to only keep only one `getProbes` and `getChildren` (with the `const`)?